src/stylesheets/variables.scss

```diff
@@ -15,6 +15,7 @@ $current-task-header-background: #2b2a3e;
 $current-task-content-background: rgba(46, 45, 64, 0.7);
 
 $task-checkbox-color: #c4c4c4;
+$fullscreen-checkbox-color: rgba(#999999, 0.33);
```

Comment:
I think this name should reference the timer, e.g. ```$timer-fullscreen-checkbox-color```. Otherwise it's a little difficult to interpret what it's for